vyper/parser/external_call.py

```diff
@@ -29,7 +29,9 @@ def external_contract_call(node,
                            pos,
                            value=None,
                            gas=None):
-    from vyper.parser.parser import parse_expr
+    from vyper.parser.expr import (
```

Comment:
is there any reason not to move this to a top level import?

Reply:
Expr calls external_call, creating a cyclic dependency.